cdap-api/src/main/java/co/cask/cdap/api/flow/flowlet/AbstractFlowlet.java

```diff
@@ -89,14 +185,18 @@ protected final FlowletContext getContext() {
 
   /**
    * @return {@link Class#getSimpleName() Simple classname} of this {@link Flowlet}
+   * @deprecated Use {@link AbstractFlowlet#setName} instead.
    */
+  @Deprecated
```

Comment:
getName, getDescription methods are used in the old (deprecated) configure method to get the name, description of the Flowlet. Since the configure method is being deprecated, just wanted to mark these methods as also deprecated.